_data/tags.yml

```diff
@@ -73,4 +73,5 @@ covid-19:
   url: covid19_data_portal.html
 csc:
   url: /fi_csc_assembly.html
-
+TSD:
```

Comment:
capital or small letters? So far, "nels", "covid-19" and "csc" are all small letters.